### DISCUSSION TITLE
Does the use of Form Objects prevents validation on component?

### DISCUSSION THREAD

**Discussion by whousehomestay** [livewire/livewire#8635] [1. Help] [answered]
I don't know, but $this->validate() on the component seems not working if we intialize form objects inside the component. I can't find this information on documentation, or maybe i just can't find it...

**Comment by joshhanley** [comment]
@whousehomestay should work fine. Can you share what your code looks like?

**Reply by whousehomestay** [reply, marked as answer]
Whops my bad, forget to show some errors, so i think there's no something working on the page. My bad... Will close this problem. Thank you for the answer josh... 🙏

**Reply by joshhanley** [reply]
All good, no worries!
